### PR TITLE
Frontend checkbackup fix

### PR DESCRIPTION
**Pull request by thisconnect**
2 small ui improvements

- show check backup title
- confirm check-backup-success and close with enter key

also refactored backkupitem to functional component

### CODE REVIEW

File: frontends/web/src/routes/device/bitbox02/checkbackup.tsx [hunk 1]
```diff
@@ -46,66 +46,70 @@ class Check extends Component<Props, State> {
...
+                  autoFocus
+                  disabled={!userVerified}
+                  primary
+                  type="submit">
+                  { userVerified ? t('button.ok') : t('accountInfo.verify') }
+                </Button>
+              )}
             </DialogButtons>
-          </div>
+          </form>
```
Comment (thisconnect) [inline frontends/web/src/routes/device/bitbox02/checkbackup.tsx:112]: changing to a form that has onSubmit event listener, and button type submit with autofocus, allows for confirming the dialog with pressing "Enter"

File: frontends/web/src/routes/device/bitbox02/checkbackup.tsx [hunk 2]
```diff
@@ -46,66 +46,70 @@ class Check extends Component<Props, State> {
...
+      const foundBackup = backups.find((backup: Backup) => backup.id === backupID);
       if (!foundBackup) {
-        alertUser(this.props.t('unknownError', { errorMessage: 'Not found' }));
+        alertUser(t('unknownError', { errorMessage: 'Not found' }));
         return;
       }
       this.setState({
-        foundBackup,
         activeDialog: true,
-        message: this.props.t('backup.check.success'),
```
Comment (thisconnect) [inline frontends/web/src/routes/device/bitbox02/checkbackup.tsx:59]: this was wrong, it did set the success title too early. moved to the end after `await bitbox02API.checkBackup(deviceID, false);`

**Comment by thisconnect** [comment]
rebased

File: frontends/web/src/routes/device/components/backup.tsx [hunk 3]
```diff
@@ -15,60 +15,60 @@
...
-}
-
-const TranslatedBackupsListItem = translate()(BackupsListItem);
-export { TranslatedBackupsListItem as BackupsListItem };
+  return radio ? (
+    <Radio
+      disabled={!!disabled}
+      checked={selectedBackup === backup.id}
+      onChange={(event: React.SyntheticEvent) => {
+        handleChange && handleChange((event.target as HTMLInputElement).value);
```
Comment (shonsirsha) [inline frontends/web/src/routes/device/components/backup.tsx:57]: I believe we could have `React.ChangeEvent<HTMLInputElement>` here so there'd be no need to cast it later on as `HTMLInputElement`, and we could directly use it like:

`event.target.value`

**Review by shonsirsha — APPROVED** [review]
lgtm w/ super small nit :)

File: frontends/web/src/routes/device/components/backup.tsx (hunk 3 above)
Comment (thisconnect) [reply frontends/web/src/routes/device/components/backup.tsx]: yes that is much nicer 👍

**Pull request merged**